numpy/core/tests/test_arrayprint.py

```diff
@@ -188,10 +189,11 @@ def test_structure_format(self):
         assert_equal(
             np.array2string(A),
             textwrap.dedent("""\
-            [('1970-01-01T00:00:00',) ('1970-01-01T00:00:00',) ('1970-01-01T00:00:00',)
-             ('1970-01-01T00:00:00',) ('1970-01-01T00:00:00',) (                'NaT',)
-             (                'NaT',) (                'NaT',) (                'NaT',)
-             (                'NaT',)]""")
```

Comment:
This is only 74 characters, but if the last line had 2 more items, it would be 76, which would be too long. It's easier to play it safe, rather than being greedy and having to backtrack.